code/game/machinery/vending.dm

```diff
@@ -640,7 +640,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/reagent_containers/food/drinks/soda_cans/cola = 8, /obj/item/reagent_containers/food/drinks/soda_cans/sodawater = 15,
 					/obj/item/reagent_containers/food/drinks/drinkingglass = 30, /obj/item/reagent_containers/food/drinks/ice = 10,
 					/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass = 12, /obj/item/reagent_containers/food/drinks/flask = 3,
-					/obj/item/reagent_containers/food/drinks/beer = 6)
+					/obj/item/reagent_containers/food/drinks/beer = 6, /obj/item/reagent_containers/food/drinks/bottle/sake = 5)

```

Comment:
add `//CIT CHANGE - adds sake to boozeomat` here, or modularize the change

Reply:
Done!